src/Api/SecretManagerFeatures/Models/Request/AccessTokenCreateRequestModel.cs

```diff
@@ -28,7 +28,7 @@ public ApiKey ToApiKey(Guid serviceAccountId)
             Name = Name,
             Key = Key,
             ExpireAt = ExpireAt,
-            Scope = "api.secrets",
+            Scope = "[\"api.secrets\"]",
```

Comment:
We'll need to figure out how to implement read/write access later, probably with two scopes.